src/Sarif/Core/Run.cs

```diff
@@ -19,7 +19,13 @@ public partial class Run
 
         public Uri GetExpandedUriBaseIdValue(string key, string currentValue = null)
```

Comment:
>GetExpandedUriBaseIdValue [](start = 19, length = 25)

Suggested method name: `ExpandUriFromUriBaseId`. Because:
1. The operation you're doing is "expanding".
2. The thing you're expanding is a uri, not a uriBaseId.
3. The word "value" at the end adds nothing.

Reply:
Renamed to `ExpandUrisWithUriBaseId`.

---
In reply to: [247330509](https://github.com/Microsoft/sarif-sdk/pull/1210#discussion_r247330509) [](ancestors = 247330509)